src/Forms/InputLabel.tsx

```diff
@@ -93,7 +104,7 @@ const StyledInputLabel = styled.label<StyledInputLabelProps>`
 `;
 
 const StyledInputLabelText = styled.span<StyledInputLabelTextProps>`
-  display: ${({ isLabelVisible }) => (isLabelVisible ? 'inline' : 'none')};
+  ${({ isLabelVisible }) => (isLabelVisible ? 'display: inline;' : accessibility)};
```

Comment:
I actually don't think we need to have `display: inline` as part of this -- that's the default display value for a span anyway, and that value shouldn't affect the changes made by the other css snippet. We should be able to leave it out entirely and just have something like

```ts
  ${({ isLabelVisible }) => (isLabelVisible ? accessibility : '')};
```

Reply:
ok